truss-chains/examples/streaming/streaming_chain.py

```diff
@@ -49,6 +49,8 @@ async def run_remote(self) -> AsyncIterator[bytes]:
             )
             print("Yield")
             yield streamer.yield_item(data)
+            if cause_error and i > 2:
```

Comment:
@joostinyi This is not written by me, but seems an odd style to have a variable that is not default that causes an error?

Reply:
Good point, it's a bit dirty to have this here in `/examples`. I'll move it into `/tests` in a follow up. But not blocking the release.